### PR TITLE
chore/refactor: rework issue_templates

### PR DESCRIPTION
**Pull request by yqlbu**
### Background

Replicate https://github.com/daeuniverse/dae/pull/267

Use new issue_templates.

Preview: https://github.com/daeuniverse/dae-1/issues/new/choose

### Checklist

- [x] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/daed

### Full changelogs

- chore/refactor: rework issue_templates

### Issue reference

Closes #_[issue number]_

### Test Result

image

image

### CODE REVIEW

File: wing
Comment (kunish) [inline wing:1]: Is this on purpose or a mistake?

Comment (yqlbu) [reply wing:1]: It was an accident...

Comment (yqlbu) [reply wing:1]: Resolved.

**Comment by yqlbu** [comment]
> Lots good stuff. Is this PR ready to go?

Yes, let’s merge it.

**Pull request merged**